packages/action/package.json

```diff
@@ -13,7 +13,8 @@
     "@actions/core": "^1.2.4",
     "@actions/github": "^3.0.0",
     "node-fetch": "^2.6.0",
-    "tap-parser": "^10.0.1"
+    "tap-parser": "^10.0.1",
+    "url": "^0.11.0"
```

Comment:
actually  no  need  for  this dep, `url` is built into Node.js  with the exact same method signature.

Reply:
resolved